documentation.md

```diff
@@ -19,11 +19,11 @@ For previous versions of the specification, please see the [Specification Links]
 Meta-schemas
 ------------
 
-The meta-schemas are the schemas which define the JSON Schema and Hyper-Schema formats.
-
+The meta-schemas define the JSON Schema and Hyper-Schema formats using the JSON Schema format.
```

Comment:
Pushed a new commit taking your suggestion other than making it "against which other schemas **can be** validated" instead of **"against which** other schemas **are** validated"